### DISCUSSION TITLE
Some guidance needed

### DISCUSSION THREAD

**Discussion by atv2016** [HandyHat/ha-hildebrandglow-dcc#199] [General]
Hi,
I'm new to these integrations:

From this repo's main readme.md:
Although this integration will work with their hardware, you should use the MQTT version [here](https://github.com/unlobito/ha-hildebrandglow/) to get real-time consumption data.

What does this mean? 
Why should i use the MQTT version if this one supports their hardware? 
And if i do, do i use the MQTT version in addition to this version ?

I would like to get realtime usage, do i have to buy their CAD for 60,- (happy to do so) and would it work with my smart meter? It's a bit annoying their CAD almost looks exactly the same as the one i already have, albeit a bit wider. I tried setting it up via the Bright app but it doesn't seem to recognise my meter (Elster Themis Alpha).

**Comment by HandyHat** [comment]
Ah, it is a little unclear as to which integration to use and why - I will look into clarifying this in the readme for future users.

This integration assumes you don't have Hildebrand's hardware, and have instead set up your meters in the Bright app getting the data via the DCC instead. The DCC only updates the data once every half an hour, and so the integration will only poll Hildebrand every half an hour. This would therefore negate the ability to get real-time data, which is why using the MQTT version is recommended instead. You would not need this integration if you install the MQTT integration.

To get real-time usage, you would need to buy their hardware or have other compatible hardware that is capable of providing real-time data. What CAD do you have? - some of them do support real-time data. There is discussion about this here: https://community.home-assistant.io/t/uk-smart-energy-meters/16545/99

If you tried to set up the meter(s) via the Bright app and it did not work, I would recommend contacting Hildebrand's brilliant support (https://shop.glowmarkt.com/pages/customer-service) before buying their hardware, as they will be able to give you further pointers.
